src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -3838,17 +3838,40 @@ private static bool CanReuseParameter(CSharp.Syntax.ParameterSyntax parameter)
                 type = null;
                 name = this.EatToken(SyntaxKind.ArgListKeyword);
             }
-
+            SyntaxToken exclamation;
+            SyntaxToken equals;
+            if (this.CurrentToken.Kind == SyntaxKind.ExclamationToken)
+            {
+                exclamation = this.EatToken(SyntaxKind.ExclamationToken);
+                equals = null;
+            }
+            else if (this.CurrentToken.Kind == SyntaxKind.ExclamationEqualsToken)
+            {
+                var notEq = this.EatToken(SyntaxKind.ExclamationEqualsToken);
+                equals = ConvertToMissingWithTrailingTrivia(notEq, SyntaxKind.EqualsToken);
```

Comment:
i don't think this is right.  the token isn't missing.  and by doin this we'll add the 'missing' flag to it.

Reply:
i'm trying to find code that shows what you should do here instead :)

Reply:
Oh, actually, this may be write.  Since you need a place to store the `!=` text onto... so we're representing this as an actual missing `=` with that trivia.  That's likely all ok.  So ignore everything above.  Sorry!

Reply:
Sounds good!